src/ReservationItem.php

```diff
@@ -554,6 +554,21 @@ public static function showListSimple()
             'display_emptychoice' => true,
         ]);
 
+        echo "</td></tr>";
+
+        // Location dropdown
+        $user = new User();
+        $user->getFromDB(Session::getLoginUserID());
+        $entities = Profile_User::getUserEntities($user->fields['id'], true);
+        $locrand = mt_rand();
+        echo "<tr class='tab_bg_1'><td><label for='dropdown_locations_id$locrand'>" . __('Item location') . "</label></td><td>";
+        Location::dropdown([
+            // Fill with submitted data if any, otherwise use user's location
+            'value'  => $_POST['locations_id'] ?? $user->fields['locations_id'],
```

Comment:
Any preselected value will not be reflected in the default list, and users may not understand why default list is not filtered by preselected value.
```suggestion
            'value'  => $_POST['locations_id'] ?? 0,
```

Reply:
The current behavior is already a default list not filtered by pre-selected/filled values of other fields (date/hour of today, for a 1h duration).

My idea was to make use of the user location to ease it's tasks.

Reply:
OK, so if it matches current behaviour, you can keep this predefined value.